it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala

```diff
@@ -34,88 +50,94 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
       ping.status.shouldBe(200)
 
       val result = callEndpoint(
-        s"""{"nino":"$NINO_VALID","givenName":"Lawrence","familyName":"Velazques","dateOfBirth":"1954-10-04", $statusRange}""")
+        s"""{"nino":"$NINO_VALID","givenName":"Lawrence","familyName":"Velazques","dateOfBirth":"1954-10-04", $statusRange}"""
```

Comment:
lots of names in here too which need updating

Reply:
done